frontends/PyCDE/test/polynomial.py

```diff
@@ -59,7 +60,8 @@ class Polynomial(pycde.System):
   def build(self, top):
     i32 = types.i32
     x = hw.ConstantOp.create(i32, 23)
-    poly = PolynomialCompute([62, 42, 6], x=x)
+    poly = PolynomialCompute("example", [62, 42, 6], x=x)
+    # PolynomialCompute("example", [62, 42, 6], x=poly.y)
```

Comment:
remove?

Reply:
I figure that's your test case. Uncomment, and it crashes without de-dup.

Reply:
Oh! It crashes? I was hoping it would just emit duplicates... Anyway, I'm digging in now, and will start there.

Reply:
Defining two modules of the same signal emits an error message... I used the term "crash" rather loosely.